src/mod/docs/index.md

```diff
@@ -31,7 +35,7 @@
 | Alumni Microsoft employees                                                                   | Stop being a Microsoft employee                                       | Decorative                                                                                                                                                                        |
 | Alumni moderator                                                                             | Stop being a moderator                                                | Decorative                                                                                                                                                                        |
 | 💙 Edgevatars Event participant                                                              | Take part in the 2019 Edgevatars Event                                | Decorative                                                                                                                                                                        |
-| Coffee Club participant                                                                      | Take part in the 2020 Coffee Club Event                               | Decorative                                                                                                                                                                        |
+| ☕ Coffee Club participant                                                                      | Take part in the 2020 Coffee Club Event                               | Decorative                                                                                                                                                                        |
```

Comment:
The coffee cup is the role icon, not a part of the role name, so it's not included here.